azext_iot/central/params.py

```diff
@@ -150,6 +150,13 @@ def load_central_arguments(self, _):
             " A case-sensitive string (up to 128 characters long) of ASCII 7-bit alphanumeric characters plus"
             " certain special characters: - . + % _ # * ? ! ( ) , : = @ $ '",
         )
+    with self.argument_context("iot central device manual-failover") as context:
+        context.argument(
+            "ttl_minutes",
+            type=int,
+            options_list=["--ttl-minutes", "--ttl"],
+            help="A positive integer. TTL in minutes to move device back to the original hub.",
```

Comment:
We should add information here that there is a default value, even if the client doesn't know about it, we should document it publically or it may customer confusion.

e.g. if a customer doesn't set this, they might expect the failover to never expire

Reply:
something like "... Has default value in backend. Client may become out of sync, see docs here on what the latest backend default time to live is aka.ms/link-here"